main/src/org/destinationsol/game/chunk/ChunkFiller.java

```diff
@@ -247,9 +247,9 @@ private void fillFarJunk(SolGame game, Vector2 chCenter, RemoveController remove
         TextureManager textureManager = game.getTexMan();
 
         for (int i = 0; i < count; i++) {
-            // Select a random far junk texture
+            // Select a random far junk atlas
```

Comment:
This reads a little funny. Made sense with "texture" but less so with "atlas" since it indicates sort of a potential collection? Not that a texture couldn't also be a collection if it has multiple things. Maybe something like "Select a random entry from the far junk atlas"